src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Open source bot built by and for the Camp Buddy Discord Fan Server.
+ *     Copyright (C) 2020  Kyuuto-devs
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.yuutoproject.yuutobot.commands
+
+import io.github.yuutoproject.yuutobot.commands.base.AbstractCommand
+import io.github.yuutoproject.yuutobot.commands.base.CommandCategory
+import io.github.yuutoproject.yuutobot.extensions.applyDefaults
+import io.github.yuutoproject.yuutobot.utils.httpClient
+import io.github.yuutoproject.yuutobot.utils.jackson
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
+import net.dv8tion.jda.api.utils.MarkdownSanitizer
+import okhttp3.Request
+import java.io.File
+import java.util.*
+
+class Owoify : AbstractCommand(
+    "owoify",
+    CommandCategory.FUN,
+    // TODO: Change this description
+    "Yuuto can turn owoify your text! (whatever that means)",
+    "owoify [level] <text>"
+) {
```

Comment:
Add aliases, as having `owoify` as the only way to call the command might annoy some people.
```suggestion
) {
    override val aliases = arrayOf("owo", "uwu")

```

Reply:
uwu doesn't make sense as alias